tests/zkevm/test_worst_compute.py

```diff
@@ -741,6 +741,101 @@ def test_worst_unop(
     )
 
 
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize("key_mut", [True, False])
+@pytest.mark.parametrize("val_mut", [True, False])
+def test_worst_tload(
+    state_test: StateTestFiller,
+    pre: Alloc,
+    key_mut: bool,
+    val_mut: bool,
+):
+    """Test running a block with as many TLOAD calls as possible."""
```

Comment:
As many TLOADs is only when key/value mutations are false, I think here a mention to figure out the zk cycles necessary for each scenario where each of these scenarios is also a stress test (mutating keys/values or not) would be fine (although the zk cycles stuff is likely implicit because this is in the zkevm dir?)

Reply:
Yup, sounds good!